kilkari/src/main/java/org/motechproject/nms/kilkari/service/DeactivationAuditService.java

```diff
@@ -0,0 +1,11 @@
+package org.motechproject.nms.kilkari.service;
+
+import org.motechproject.nms.kilkari.domain.SubscriptionOrigin;
+import org.motechproject.nms.kilkari.domain.SubscriptionStatus;
+
+public interface DeactivationAuditService {
+
+    void auditSuccess(String subscriptionId, Long subscriberId, SubscriptionOrigin subscriptionOrigin, SubscriptionStatus status, Long msisdn);
```

Comment:
Design suggestion: given that these two methods are just calling the same add audit method, why not have an enum field in the deactivationSubscriptionRecord with indicates success and failure with another column to record the error string if any?

It feels like we are adding complexity by adding two separate method interfaces and while an audit query could be as simple as select * from foo where column=success, we are making it harder to query with contains error string like '%'.

Again, it is a suggestion and you have the ownership to say no.